make/tasks/shared/lint/lockfiles.sh

```diff
@@ -1,9 +1,9 @@
 #!/bin/bash
 
-set -e
+set -euo pipefail
```

Comment:
`set -u` treats not set environment variables as errors. It helps catch misspellings and other common errors, which might otherwise go unnoticed.
`set -o pipefail` ensures that if you have a command pipeline (e.g., cmd1 | cmd2 | cmd3), the pipeline’s return status is the value of the last (rightmost) command to exit with a non-zero status, or zero if all commands exit successfully. Without this setting, a pipeline returns the exit code of the last command in the pipeline, potentially masking failures of previous commands.